rakelib/dependency.rake

```diff
@@ -1,7 +1,7 @@
 
 namespace "dependency" do
   task "bundler" do
-    Rake::Task["gem:require"].invoke("bundler", "~> 1.9.4")
+    Rake::Task["gem:require"].invoke("bundler", "~> 1")
```

Comment:
@jsvd maybe just make this `1.16.0` exactly like we have in the integration tests? This isn't really a dependency that must be free floating (imo) ( `~> 1` seems wrong because we now changed the API calls in a way that will only work with newer `Bundler`?).